pkg/minishift/network/settings_windows.go

```diff
@@ -41,10 +42,13 @@ var (
 )
 
 func isUsingDefaultSwitch() bool {
-	switchEnvName := "HYPERV_VIRTUAL_SWITCH"
-	switchEnvValue := os.Getenv(switchEnvName)
+	posh := powershell.New()
+	switchNameCommand := fmt.Sprintf("$swname=(Get-VM \"%s\" | Get-VMNetworkAdapter | select SwitchName).SwitchName", profile.GetActiveProfile())
```

Comment:
`get-vm`? This would mean this only works AFTER the vm has been created, right?

Reply:
@gbraad Right, this function is used in the process of setting the ip address which happens after the vm has been created so, and also the name suggests, if the VM is using the default switch